sidekick_vault/test/decrypt_test.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:sidekick_core/sidekick_core.dart';
+import 'package:sidekick_vault/sidekick_vault.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('decrypt:', () {
+    test('no file', () {
+      final runner = CommandRunner('', '')..addCommand(VaultCommand());
+      expect(
+        () => runner.run(['vault', 'decrypt']),
+        throwsA(
+          isA<String>().having(
+            (it) => it,
+            'error',
+            'Missing file'
+                '\n'
+                'Example:\n'
+                'noa vault decrypt file.csv.gpg',
+          ),
+        ),
+      );
+    });
+    test('not a file', () {
+      final runner = CommandRunner('', '')..addCommand(VaultCommand());
+      expect(
+        () => runner.run(['vault', 'decrypt', '.']),
+        throwsA(
+          isA<String>().having(
+            (it) => it,
+            'error',
+            'No valid file'
+                '\n'
+                'Example:\n'
+                'noa vault decrypt file.csv.gpg',
```

Comment:
Idea: Instead of matching the exact String you could only match that the String contains the message and the example separately. That would make the test resilient towards error formatting changes